lib/config.go

```diff
@@ -459,7 +462,19 @@ func (c *RuntimeConfig) Validate(onExecution bool) error {
 
 	// check we do have at least a runtime
 	if _, ok := c.Runtimes[c.DefaultRuntime]; !ok {
-		return errors.New("no default runtime configured")
+		// Set the default runtime to "runc" if default_runtime is not set
+		if c.DefaultRuntime == "" {
+			logrus.Warnf("Defaulting to `runc` as the runtime since default_runtime is not set")
```

Comment:
logrus.Warnf("Defaulting to `%s` as the runtime since default_runtime is not set", defaultRuntime)

Reply:
Fixed!